encrypted-media-respec.html

```diff
@@ -2331,15 +2365,16 @@
 
         <section id="queue-waitingforkey">
           <h4>Queue a "waitingforkey" Event</h4>
-          <p>The Queue a "waitingforkey" Event algorithm is run when the CDM cannot decrypt the current block.
+          <p>The Queue a "waitingforkey" Event algorithm is run when the CDM cannot decrypt an encrypted block.
             It should only be called when the <a def-id="htmlmediaelement"></a> object is <a def-id="potentially-playing"></a> and its <a def-id="readystate"></a> is equal to <a def-id="have-future-data"></a> or greater.  
             Requests to run this algorithm include a target <a def-id="htmlmediaelement"></a> object.
           </p>
           <p>The following steps are run:</p>
           <ol>
             <li><p>Let the <var>media element</var> be the specified <a def-id="htmlmediaelement"></a> object.</p></li>
-            <li><p>If the <var>media element</var>'s <var>waiting for key</var> value is <code>false</code>, <a def-id="queue-a-task-to-fire-an-event-named"></a> <a def-id="waitingforkey"></a> at the <var>media element</var>.</p></li>
+            <li><p>If the <var>media element</var>'s <var>waiting for key</var> value is <code>true</code>, abort these steps.
             <li><p>Set the <var>media element</var>'s <var>waiting for key</var> value to <code>true</code>.</p></li>
```

Comment:
Should we add a Note that the media element is now a "blocked media element" and thus not "potentially playing"?
(I suggested this above where this algorithm is called. It is the setting of `<var>waiting for key</var>` to true that triggers this, though. Perhaps we should move all these steps to the call point. This would also address my comment immediately above.

The other use of this algorithms is in the Initialization Data Encountered algorithm. I believe the state and Note are appropriate for that path as well, so maybe we should keep/put them here. We may want to rename this algorithm, though.